package/build-and-ship.sh

```diff
@@ -21,13 +16,8 @@ export VERSION
 cd "$(dirname "$0")"
 ./build-package.sh
 
-if [[ "$SEMAPHORE_GIT_REF_TYPE" != "tag" ]]; then
-    echo "Not on a tagged build. Skipping ship step."
-    exit 0
-fi
-
 PKGFILE="hoff_$VERSION-1.deb"
-FREIGHT_HOST="freight@archive-external.channable.com"
+FREIGHT_HOST="archive-petrol"
 
 scp "$PKGFILE" "$FREIGHT_HOST:/tmp/$PKGFILE"
```

Comment:
This does not work as-written, I think. `gcloud compute scp --tunnel-through-iap` would be required. Similarly for the SSH command below.

Reply:
It does work for people who've provisioned `gcloud_ssh_config` on their machine :p
I'll change it though (and also the `scp` line)